modules/persistent-storage-local-tolerations.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-tolerations_{context}"]
+= Using tolerations with Local Storage Operator Pods
+
+You can control which nodes the Local Storage Operator Pod runs on and prevent
+other workloads from using those nodes by using tolerations on the Pods.
+
+You apply tolerations to the Local Storage Operator Pod through the LocalVolume Custom Resource (CR)
```

Comment:
Replace `Custom Resource (CR)` with `resource`. This may be a preference, but `Custom Resource (CR)` is an implementation detail that the user doesn't need to know.